dags.yaml

```diff
@@ -1179,3 +1179,24 @@ bqetl_default:
   tags:
     - impact/tier_3
     - triage/no_triage
+
```

Comment:
Does this need to be in a separate DAG from `bqetl_monitoring`?

Reply:
probably not, I will move it

Reply:
DAG moved